File: tourneyAPI/Models/Entities/Characters/Simon.cs
namespace Entities;

using Enums;

// Defines the competitive profile metadata for this playable character.
public class Simon : Character
{
    // Initializes this character's default competitive attributes for matchmaking and tier logic.
    public Simon()
    {
        Id = CharacterId.Simon;
        characterName = CharacterName.SIMON;
        archetype = Archetype.TURTLE;
        fallSpeed = FallSpeed.FAST_FALLERS;
        weightClass = WeightClass.MIDDLEWEIGHT;
        tierPlacement = TierPlacement.D;
    }

}

File: tourneyClient/src/models/entities/Characters/Simon.ts
import { CharacterId } from "@/models/Enums/CharacterId";
import { Character } from "../Character"
import { CharacterName } from "@/models/Enums/CharacterName";
import { Archetype } from "@/models/Enums/Archetype";
import { FallSpeed } from "@/models/Enums/FallSpeed";
import { WeightClass } from "@/models/Enums/WeightClass";
import { TierPlacement } from "@/models/Enums/TierPlacement";

export const Simon: Character =
{
    id: CharacterId.Simon,
    characterName: CharacterName.SIMON,
    archetype: Archetype.TURTLE,
    fallSpeed: FallSpeed.FAST_FALLERS,
    weightClass: WeightClass.MIDDLEWEIGHT,
    tierPlacement: TierPlacement.D
}

export default Simon; 